XamarinCommunityToolkit/Behaviors/EffectBehavior.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public class EffectBehavior : BaseBehavior<View>
+    {
+        public static readonly BindableProperty GroupProperty = BindableProperty.Create(nameof(Group), typeof(string), typeof(EffectBehavior));
+        public static readonly BindableProperty NameProperty = BindableProperty.Create(nameof(Name), typeof(string), typeof(EffectBehavior));
+
+        public string Group
+        {
+            get => (string)GetValue(GroupProperty);
+            set => SetValue(GroupProperty, value);
+        }
+
+        public string Name
+        {
+            get => (string)GetValue(NameProperty);
+            set => SetValue(NameProperty, value);
+        }
+
+        protected override void OnAttachedTo(BindableObject bindable)
+        {
+            base.OnAttachedTo(bindable);
+            AddEffect(bindable as View);
+        }
+
+        protected override void OnDetachingFrom(BindableObject bindable)
+        {
+            RemoveEffect(bindable as View);
+            base.OnDetachingFrom(bindable);
+        }
+
+        void AddEffect(View view)
+        {
+            var effect = GetEffect();
+
+            if (effect == null)
+            {
+                return;
+            }
+
+            view.Effects.Add(GetEffect());
+        }
+
+        void RemoveEffect(View view)
+        {
+            var effect = GetEffect();
+
+            if (effect == null)
+            {
+                return;
+            }
+
+            view.Effects.Remove(GetEffect());
+        }
+
+        Effect GetEffect()
+        {
+            if (!string.IsNullOrWhiteSpace(Group) && !string.IsNullOrWhiteSpace(Name))
+            {
+                return Effect.Resolve(string.Format("{0}.{1}", Group, Name));
```

Comment:
Maybe use the newer string formatting $"".

Reply:
What Glenn said ☝️